doc/user/content/release-notes/_index.md

```diff
@@ -7,8 +7,41 @@ menu:
     name: "Release notes"
     identifier: "release-notes"
 ---
+## v25.2.8
+
+### Bug fixes
+- Fix bug where users could not login after upgrading to version v25.2.6 or v25.2.7.
+
+## v25.2.7
+
+{{< warning >}}
+This version may cause unintended password resets when using password authentication. We recommend upgrading to v25.2.8+.
+{{</ warning>}}
+
+### Enable multi-replica source clusters
+Clusters with sources can now have multiple sources by default.
+
+### Bug fixes
+- Fix bug where role modifications can wipe out role password.
+- Fix bug where if two Materialize instances are in the same namespace and have clusters with the same ID, their services could query the wrong one.
+- Fix bug where when restarting a snapshot when adding a table to a source via `ALTER SOURCE`, we wouldn't cancel the existing snapshotting process.
```

Comment:
```suggestion
- Fix bug where restarting a snapshot while adding a table to a source via `ALTER SOURCE` would cause Materialize to not cancel the existing snapshotting process.
```